src/common/transformations/include/transformations/utils/utils.hpp

```diff
@@ -19,6 +19,16 @@
 #include "transformations/rt_info/attributes.hpp"
 #include "transformations_visibility.hpp"
 
+/* This macro is intended to fix C++20 [=] lambda
+warning. Although C++20 identifier is 202002L,
+some compilers supporting C++20, or its drafts like
+C++2a, producing the warning, are using 201402L value. */
+#if __cplusplus >= 201402L
+#    define OV_CAPTURE_ALL_VAL =, this
+#else
+#    define OV_CAPTURE_ALL_VAL =
```

Comment:
Just maybe reconsider naming of this macro, as `=` means capture by copy but using `this` is capture by reference.

Reply:
I also don't like this name and I'll try to come up with something else. Any suggestions?